tachyon/zk/base/blinder.h

```diff
@@ -28,12 +28,17 @@ class Blinder {
   }
   size_t blinding_factors() const { return blinding_factors_; }
 
-  // Blinds |evals| at behind by |blinding_factors_|.
-  // Returns false if |evals.NumElements()| is less than |blinding_factors_|.
-  bool Blind(Evals& evals) {
+  // The number of |blinding_rows| is determined to be either
+  // |blinding_factors_| or |blinding_factors_| + 1, depending on the
+  // |include_boundary_row| option.
+  // Blinds |evals| at behind by |blinding_rows|.
+  // Returns false if |evals.NumElements()| is less than |blinding_rows|.
+  bool Blind(Evals& evals, bool include_boundary_row = false) {
```

Comment:
Please rename `boundary` to `last` as per https://zcash.github.io/halo2/design/proving-system/permutation.html#zero-knowledge-adjustment